code/modules/recycling/disposal.dm

```diff
@@ -225,6 +225,9 @@
 	if(!check_mob_size(target))
 		to_chat(user, SPAN_NOTICE("The opening is too narrow for [target] to fit!"))
 		return
+	if(target.isSynthetic())
```

Comment:
This applies to every human mob whos external organs are completely robotic.
If the goal is to prevent the flushing of borgs, then this implementation is invalid as it also prevents the flushing of other unrelated mobs.

Either this needs to be changed or clarified that it applying to IPCs is intentional and not a oversight.

Reply:
Roger